doc/examples/tracking_tissue_classifier.py

```diff
@@ -0,0 +1,184 @@
+model.
+"""
+import numpy as np
+
+from dipy.data import read_stanford_labels
+from dipy.reconst.csdeconv import ConstrainedSphericalDeconvModel
+from dipy.tracking import utils
+
+hardi_img, gtab, labels_img = read_stanford_labels()
+data = hardi_img.get_data()
+labels = labels_img.get_data()
+affine = hardi_img.get_affine()
+
+seed_mask = labels == 2
+white_matter = (labels == 1) | (labels == 2)
+seeds = utils.seeds_from_mask(seed_mask, density=1, affine=affine)
+
+csd_model = ConstrainedSphericalDeconvModel(gtab, None, sh_order=6)
+csd_fit = csd_model.fit(data, mask=white_matter)
+
+"""
+-- ThresholdTissueClassifier --
+A scalar map can be used to build a tissue classifier by thresholding it to a
+fixed value. Here, we show an example using the fractional anisotropy (FA) map
+of the DTI model. The threshold tissue classifier uses a trilinear
+interpolation method at the tracking position.
+
+Parameters:
+    metric_map: numpy array [:,:,:],
+    threshold: float.
+
+'ENDPOINT': metric_map < threshold,
+'OUTSIDEIMAGE': tracking point outside of metric_map,
+'TRACKPOINT': stopped because no direction are available,
+'INVALIDPOINT': N/A.
+"""
+
+import dipy.reconst.dti as dti
+from dipy.reconst.dti import fractional_anisotropy
+from dipy.tracking.local import ThresholdTissueClassifier
+
+tensor_model = dti.TensorModel(gtab)
+tenfit = tensor_model.fit(data, mask=white_matter)
+FA = fractional_anisotropy(tenfit.evals)
+
+threshold_classifier = ThresholdTissueClassifier(FA, .2)
+
+"""
+-- BinaryTissueClassifier --
+A binary mask can be used as tissue classifier for the tractography.
+Here, we show how to obtain the binary tissue classifier from
+the white matter mask defined above. The binary tissue classifier uses a
+nearest-neighbourhood interpolation method at the tracking position.
+
+Parameters:
+    mask: numpy array [:,:,:],
+
+'ENDPOINT': mask = 0,
+'OUTSIDEIMAGE': tracking point outside of mask,
+'TRACKPOINT': stopped because no direction are available,
+'INVALIDPOINT': N/A.
+"""
+from dipy.tracking.local import BinaryTissueClassifier
+
+binary_classifier = BinaryTissueClassifier(white_matter)
+
+"""
+-- ActTissueClassifier --
+Anatomically-constrained tractography (ACT) uses information from
+anatomical images to determine when the tractography stops. The 'include_map'
+defines when the streamline reached a 'valid' stopping region (e.g. gray
+matter partial volume estimation (PVE) map) and the 'exclude_map' defines when
+the streamline reached an 'invalid' stopping region (e.g. corticospinal fluid
+(PVE) map). The background of the anatomical image should be added to the
+'include_map' to keep streamline exiting the brain (e.g. throught the
+brain stem). The act tissue classifier uses a trilinear interpolation method
+at the tracking position.
+
+Parameters:
+    include_map: numpy array [:,:,:],
+    exclude_map: numpy array [:,:,:],
+
+'ENDPOINT': include_map > 0.5,
+'OUTSIDEIMAGE': tracking point outside of metric_map,
```

Comment:
But there is no parameter `metric_map` here. Should this be one of `include_map`, 'exclude_map', or is there an additional input?